x-pack/plugins/observability_solution/observability/public/pages/alert_details/components/alert_summary.tsx

```diff
@@ -18,15 +18,18 @@ interface AlertSummaryProps {
 
 export function AlertSummary({ alertSummaryFields }: AlertSummaryProps) {
   return (
-    <EuiFlexGroup data-test-subj="alert-summary-container" gutterSize="xl">
-      {alertSummaryFields?.map((field, idx) => {
-        return (
-          <EuiFlexItem key={idx} grow={false}>
-            <EuiText color="subdued">{field.label}</EuiText>
-            <EuiText>{field.value}</EuiText>
-          </EuiFlexItem>
-        );
-      })}
-    </EuiFlexGroup>
+    <>
+      <EuiFlexGroup data-test-subj="alert-summary-container" gutterSize="xl">
+        {alertSummaryFields?.map((field, idx) => {
+          return (
+            <EuiFlexItem key={idx} grow={false}>
+              <EuiText color="subdued">{field.label}</EuiText>
+              <EuiText>{field.value}</EuiText>
+            </EuiFlexItem>
+          );
+        })}
+      </EuiFlexGroup>
+      {alertSummaryFields && alertSummaryFields.length > 0 && <EuiSpacer size="l" />}
```

Comment:
nit: Maybe we can check this for the whole section, like:

```
{ alertSummaryFields && alertSummaryFields.length &&
    <EuiFlexGroup data-test-subj="alert-summary-container" gutterSize="xl">
        {alertSummaryFields.map((field, idx) => {
          return (
            <EuiFlexItem key={idx} grow={false}>
              <EuiText color="subdued">{field.label}</EuiText>
              <EuiText>{field.value}</EuiText>
            </EuiFlexItem>
          );
        })}
      </EuiFlexGroup>
      <EuiSpacer size="l" />
}

Reply:
updated in 1c25061873737610b24e7a141ec555517971ed52